doc/release/1.1.0-notes.rst

```diff
@@ -29,87 +31,633 @@ New features
-'randtobest1exp', respectively. Note: These names were already in use but
-implemented a different mutation strategy. See `Backwards incompatible changes`_,
-below.
-The ``init`` keyword for the `scipy.optimize.differential_evolution` function
-can now accept an array. This array allows the user to specify the entire
-population.
-
+The method ``trust-constr`` has been added to
+`scipy.optimize.minimize`. The method switches between two
+implementations depending on the problem definition. For equality
+constrained problems it is an implementation of a trust-region
+sequential quadratic programming solver and, when inequality constraints
+are imposed, it switches to a trust-region interior point method. Both
+methods are appropriate for large scale problems. Quasi-Newton options
+BFGS and SR1 were implemented and can be use to approximate second order
+derivatives for this new method. Also, finite-differences can be used to
+approximate either first-order or second-order derivatives.
+
+Random-to-Best/1/bin and Random-to-Best/1/exp mutation strategies were
+added to `scipy.optimize.differential_evolution` as ``randtobest1bin``
+and ``randtobest1exp``, respectively. Note: These names were already in
+use but implemented a different mutation strategy. See `Backwards
+incompatible changes <#backwards-incompatible-changes>`__, below. The
+``init`` keyword for the `scipy.optimize.differential_evolution`
+function can now accept an array. This array allows the user to specify
+the entire population.
+
+Add an ``adaptive`` option to Nelder-Mead to use step parameters adapted
+to the dimensionality of the problem.
+
+Minor improvements in `scipy.optimize.basinhopping`.
 
 `scipy.signal` improvements
 ---------------------------
 
-Three new functions for peak finding in one-dimensional arrays were added.
-`scipy.signal.find_peaks` searches for peaks (local maxima) based on simple value
-comparison of neighbouring samples and returns those peaks whose properties match
-optionally specified conditions for their height, prominence, width, threshold
-and distance to each other. `scipy.signal.peak_prominences` and
-`scipy.signal.peak_widths` can directly calculate the prominences or widths of
-known peaks.
+Three new functions for peak finding in one-dimensional arrays were
+added. `scipy.signal.find_peaks` searches for peaks (local maxima) based
+on simple value comparison of neighbouring samples and returns those
+peaks whose properties match optionally specified conditions for their
+height, prominence, width, threshold and distance to each other.
+`scipy.signal.peak_prominences` and `scipy.signal.peak_widths` can directly
+calculate the prominences or widths of known peaks.
+
+Added ZPK versions of frequency transformations:
+`scipy.signal.bilinear_zpk`, `scipy.signal.lp2bp_zpk`,
+`scipy.signal.lp2bs_zpk`, `scipy.signal.lp2hp_zpk`,
+`scipy.signal.lp2lp_zpk`.
 
+Added `scipy.signal.windows.general_cosine` and
+`scipy.signal.windows.general_hamming`.
```

Comment:
and scipy.signal.windows.dpss

(#7802). I see it's mentioned in the deprecations section, but perhaps it makes sense to have it here, too.

Otherwise LGTM.